GOVERNANCE.md

```diff
@@ -182,6 +183,38 @@ The project Steward is currently: **@FredKSchott**
 - New stewards will be added based on a unanimous vote by the existing Steward(s).
 - In the event that someone is unreachable then the decision will be deferred.
 
+
+## Other Roles
+
+### Staff
+
+Staff is a special designation for employees of [The Astro Technology Company](https://astro.build/company) that lives outside of our Governance model. The staff role was designed to help those of us working full-time on Astro to work productively without "skipping the line" and circumventing our governance model entirely.
+
+The staff role was designed to offer instant **visibility** and **trust**, but not instant **authority.**
+
+#### Privileges
+
+All privileges of the [Core Maintainer role](#level-3---core-mainainer), except...
+
+- Instead of gaining Discord contributor roles (`@contributor`, `@maintainer`, `@core`) you will receive a special `@staff` role in Discord and GitHub that grants equivalent visibility and permissions as `@core`.
+- No voting abilities for the first 3 months of staff membership. Then, the role grants equivalent voting permissions as `@core` (see [Voting](#voting) below).
+- Not eligible for additional contributor levels while acting as `@staff`. You can retain all current contributor levels and can request a new nomination upon leaving `@staff` (See [Leaving Staff](#leaving-staff) below). This rule is designed to prevent a team of contributors/maintainers that is overwhelmingly ex-staff members.
```

Comment:
Could this be limited in scope, say to `@core`?

I’m unaware of any current staff who were not already maintainers before being hired. For instance, I think Tony and I were 'maintainers' beforehand, not 'core', and so I’m unsure what scenario of ex-staff members who were never contributors or maintainers that this is targeting.

However, if I imagine a future where there are whole teams of ex-staff, I am uncomfortable with this for 2 reasons. 1. Would really good interns have to start over? 2. If we became a big ol’ company and I was President and did something _really bad_ like help start a war, I would want a bunch people to be able to quit and keep amazing powers to lead the revolt against us. I may be very biased here having grown up in the US, but I like that at scale this feels like a check and balance.

Disclaimer: I am not running for President of Astro Technology Company, and I have no intention of starting wars, ever.

Reply:
haha, I like the visual. All hail our new Astro overlords :)

The idea is to move from **opt-out** ("Hey, I'm leaving staff and am opting out of the core role because I am no longer interested") to **opt-in** ("Hey, I'm leaving staff but would actively like to keep contributing!"). 

The change should have no impact on your **ability** to become a core maintainer post-staff (aka no impact on war scenario, since nominations are a core vote and not at the will of the project steward).

Reply:
I appreciate this change; I can imagine that if wanted, any staff that left that still wanted to continue to contribute and increase in role could easily progress quickly anyways...

Reply:
How is that status tracked? Just by someone belonging to `@/maintainers` and `@/staff` roles in discord and github? And then loses the `@/staff` role when leaving, giving them just `@/maintainers` role + the opportunity to become a core maintainer?

Reply:
The plan outlined here is: 
- you can be nominated for `maintainer` & `core` before joining staff
- once joining staff, you can no longer be nominated for those roles. (but keep the roles you have)
- once leaving staff, you can once more be nominated for those roles.